### PR TITLE
Added GSoD to Jsoc section

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

File: jsoc/index.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ Those participating in the program can check out [the getting started page](/lea
 Are you interested in Julia but not super familiar with the language?
 Head over to [JuliaAcademy](https://juliaacademy.com) to take some free online Julia classes.
 
+## Google Season of Docs (GSoD)
+
+The Julia organization is applying to participant in [Google's Season of Docs](https://developers.google.com/season-of-docs).
```
Comment (tlienart) [inline jsoc/index.md:31]: applying to participate ?

File: jsoc/index.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ Those participating in the program can check out [the getting started page](/lea
...
+The Julia organization generally fields a large number of students mentored by open source developers, university professors at leading institutions from all around the world, and technical writers in the Julia Community. 
+Students who are accepted into this prestigious program are paid a stipend to contribute open source documentation around  around Julia and its package ecosystem.
```
Comment (tlienart) [inline jsoc/index.md:33]: around  2x, contribute _to_ ? (maybe not)

File: jsoc/index.md (hunk 1 above)
Comment (logankilpatrick) [reply jsoc/index.md:31]: Yeah, we are not in yet, but we need to have out projects list publicly available

File: jsoc/index.md (hunk 2 above)
Comment (logankilpatrick) [reply jsoc/index.md]: Fixed this, thanks for pointing it out

File: jsoc/index.md (hunk 1 above)
Comment (tlienart) [reply jsoc/index.md:31]: oh sorry I meant `participant <> participate`

**Comment by abhishalya** [comment]
It would be really nice to have Julia participating in GSoD. Although the number of people being selected is usually low (like max 2 per org as mentioned in their FAQs).

**Comment by logankilpatrick** [comment]
Who knows what it will be like in the future though, we only have 1 session worth of data at this point. Perhaps the program will grow :)

**Pull request merged**